packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -164,19 +181,19 @@ export const Tooltip = ({
         ref={elementRef}
         className={targetWrapperClassName}
         onMouseEnter={(evt: MouseEvent) => {
-          setShow(true);
+          setIsHoveringTarget(true);
           if (onMouseOver) onMouseOver(evt);
         }}
         onMouseLeave={(evt: MouseEvent) => {
-          setShow(false);
+          setTimeout(() => setIsHoveringTarget(false), delay);
           if (onMouseLeave) onMouseLeave(evt);
-        }}
+      }}
         onFocus={(evt: FocusEvent) => {
-          setShow(true);
+          setIsHoveringTarget(true);
           if (onFocus) onFocus(evt);
         }}
         onBlur={(evt: FocusEvent) => {
-          setShow(false);
+          setTimeout(() => setIsHoveringTarget(false), delay);
```

Comment:
I think we also need setTimeout here for the keyboard short cut to work properly i.e. to focus on the button inside the tooltip box